2406495615_MadeShandyKrisnanda.py

```diff
@@ -1,112 +1,92 @@
-        t.circle(50)
-        t.penup()
-        t.forward(125)
-        t.pendown()
-    for i in range(2):
-        t.penup()
-        t.goto(-325/2,-90)
-        t.forward(125*(i+1))
-        t.pencolor((pen_color[i*6+3],pen_color[i*6+4],pen_color[i*6+5]))
-        t.pendown()
-        t.circle(50)
-        t.penup()
-        t.pendown()
-    for i in range(3):
-        t.penup()
-        t.pencolor((pen_color[i*6],pen_color[i*6+1],pen_color[i*6+2]))
-        t.home()
-        t.goto(-100, -50)
-        t.penup()
-        t.forward(125*i)
-        t.penup()
-        t.circle(50, 45)
-        t.pendown()
-        t.circle(50, 180)
-        t.penup()
-        t.circle(50, 90)
-        t.pendown()
-        t.circle(50, 45)
-        
-        
-def olympic_line():
-    t.speed(0)
+start_x = -130
+move_x = 65
+quarter_circle,half_circle, full_circle = 90, 180, 360
+onesixth_circle, oneeighth_part = 60, 45
+y_top_wave, y_bottom_wave = 120, 80
+y_line = -50
+range_box_circle = 50
+
+
+#Inputting colors according to the RGB code into variables and lists
+blue = (0,129,200)
+yellow = (252,177,49)
+black = (0,0,0)
+red = (0,166,81)
+green = (238,51,78)
+pen_color = [blue, yellow, black, red, green]
+
+def make_circle(x, y, degree, color): #Make circles
+    t.pencolor(color)
     t.penup()
     t.home()
-    t.goto(-100,-220)
+    t.goto(x, y)
     t.pendown()
+    t.circle(50, degree)
+
+def olympic_wave(): #Create an Olympic Wave logo
+    for i in range(5): 
+        if i % 2 == 0 :
+            make_circle(start_x+i*move_x,y_top_wave,full_circle,pen_color[i])
+        elif i % 2 == 1 :
+            make_circle(start_x+i*move_x,y_bottom_wave,full_circle,pen_color[i])
     for i in range(5):
-        t.pencolor((pen_color[i*3],pen_color[i*3+1],pen_color[i*3+2]))
-        t.circle(50)
-        t.penup()
-        t.forward(125/2)
-        t.pendown()
-    t.penup()
-    t.goto(-100,-220)
-    t.speed()
-    for i in range(4):
-        t.pencolor(pen_color[i*3],pen_color[i*3+1],pen_color[i*3+2])
-        t.penup()
-        t.home()
-        t.goto(-100,-220)
-        t.forward(125*(i)/2)
-        t.circle(50, 90)
-        t.pendown()
-        t.circle(50, 90)
-        t.penup()
-        t.pendown()
-def asking_box():
+        if i % 2 == 0 :
+            make_circle(start_x+i*move_x,y_top_wave,-oneeighth_part,pen_color[i])
+            make_circle(start_x+i*move_x,y_top_wave,half_circle,pen_color[i])
+        elif i % 2 == 1 :
+            make_circle(start_x+i*move_x,y_bottom_wave,-oneeighth_part*3,pen_color[i])
+            make_circle(start_x+i*move_x,y_bottom_wave, half_circle,pen_color[i])
+
+def olympic_line(): #Make a straight Olympic logo
+    for i in range(5):
+        make_circle(start_x+i*move_x,y_line, full_circle,pen_color[i])
+    for i in range(5):
+        make_circle(start_x+i*move_x,y_line,-quarter_circle,pen_color[i])
+        make_circle(start_x+i*move_x,y_line, half_circle,pen_color[i])
```

Comment:
could calculate the x first for readability and remove top bottom naming in y since it is not necessary